### DISCUSSION TITLE
Lesson 12: test reverts `DSCEngine__BreaksHealthFactor(5000)`

### DISCUSSION THREAD

**Discussion by AnouarBF** [Cyfrin/foundry-full-course-cu#1044] [Q&A]
I found some difficulties to test the liquidate function so I tried to copy the tests from the test folder. 
But all of the tests related to the liquidate function revert this error `DSCEngine__BreaksHealthFactor(5000)` So I couldn't pass the challenge of 85%

**Comment by theirrationalone** [comment]
Yessss, That's the bug @patrick was talking about.... I would suggest you to watch again from 20 minutes before the challenge. Good Luck 🍀🍀;`)
